cli/cmd/aeon.go

```diff
@@ -26,11 +30,12 @@ var connectCtx = aeoncmd.ConnectCtx{
 
 func newAeonConnectCmd() *cobra.Command {
 	var aeonCmd = &cobra.Command{
-		Use:   "connect URI",
+		Use:   "connect URI or connect PATH instance",
```

Comment:
```suggestion
		Use:   "tt aeon connect [ URI | /path/to/config INSTANCE_NAME ] [flags]",
```
Maybe something like this?